README.md

```diff
@@ -18,19 +18,17 @@ We recommend using the [latest release](https://github.com/EBIvariation/vcf-vali
 
 ### Validator
 
-vcf-validator can run on both compressed and non-compressed input VCF files.
-supported compressios are .bz2 .gz and .z extensions. For rest others pipes can be used for compressed files (see below). 
-It accepts input in the following ways:
+vcf-validator accepts both compressed and non-compressed input VCF files. Supported compression formats are .bz2, .gz and .z. For other formats such as .zip, the `zcat` command and a pipe can be used (see below).
 
-* Reading Uncompressed files:
+Reading uncompressed files:
   * File path as argument: `vcf_validator -i /path/to/file.vcf`
   * Standard input: `vcf_validator < /path/to/file.vcf`
-  * Standard input from pipe: `zcat /path/to/file.vcf.gz | vcf_validator`
+  * Standard input from pipe: `zcat /path/to/file.vcf | vcf_validator`
 
-* Reading compressed files:
-  * File path as argument: `vcf_validator -i /path/to/compressedfile.vcf.gz2`
-  * Standard input: `vcf_validator < /path/to/compressedfile.vcf.z`
-  * Standard input from pipe: `cat /path/to/compressedfile.vcf.z | vcf_validator`
+Reading compressed files:
+  * File path as argument: `vcf_validator -i /path/to/compressed_file.vcf.gz`
+  * Standard input: `vcf_validator < /path/to/compressed_file.vcf.z`
+  * Standard input from pipe: `cat /path/to/compressed_file.vcf.zip | vcf_validator`
```

Comment:
validator is not capable to read zip format directly.

Reply:
I meant to use zip only in the "uncompressed from pipe" as `zcat vcf.zip | validator`. this line (compressed + pipe) was ok as  `cat /path/to/compressedfile.vcf.z | vcf_validator`